src/Libraries/Microsoft.Extensions.Telemetry/Logging/ExtendedLogger.ModernTagJoiner.cs

```diff
@@ -56,21 +56,23 @@ public void SetIncomingTags(LoggerMessageState value)
         {
             get
             {
-                if (index < _incomingTagsCount)
+                // Keep the order of the if-else statements. It ensures that the {OriginalFormat}
```

Comment:
would smth like below bring more clarity
```suggestion
                // Iterating over _incomingTags at the end, because it contains {OriginalFormat} property which needs to be the last.
```

Reply:
Thank you, updated the comments.